src/roi/Roi.ts

```diff
@@ -292,25 +288,17 @@ export class Roi {
   }
   /**
    * Computes current ROI points.
-   * @returns Array of points. It's an array of tuples, each tuple being the x and y coordinates of the ROI point.
+   * @param kind - kind of coordinates.
+   * @returns Array of points with relative (coordinates are calculated based on ROI's point of origin) or absolute (coordinates are calculated based on the origin point of  an image/ROI map) ROI coordinates.
```

Comment:
This explains the difference between relative and coordinate and so should be explained at the level of the `kind` parameter.

Avoid making complicated sentences with a lot of content in parenthesis.